### DISCUSSION TITLE
Running on vast.ai

### DISCUSSION THREAD

**Discussion by kyledam** [TheLastBen/fast-stable-diffusion#2015] [Q&A] [answered]
Hi, How can I run it on vast.ai. I select the template [stable-diffusion:fast-stable-diffusion-2.1.0](https://hub.docker.com/r/runpod/stable-diffusion/) https://hub.docker.com/r/runpod/stable-diffusion/, everything went fine but after all, It created the link from runpod proxy. How can i make it work on vast.ai?

**Comment by TheLastBen** [comment, marked as answer]
I don't maintain vastai notebooks, only runpod and paperspace
